### PR TITLE
Member Card created

### PR DESCRIPTION
**Pull request by syedmahdi30**
Screenshot_3

Member Card

### CODE REVIEW

File: src/components/board/Member.jsx [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+  };
+
+  const imageWrapperStyle = {
+    borderRadius: "15px",
+    overflow: "hidden",
+  };
+
+  return (
+    <div style={containerStyle}>
+      <div style={iconStyle}>
```
Comment (pondohoo) [inline src/components/board/Member.jsx:42]: don't use inline css, use tailwind like u did below (use className)

File: src/components/board/Member.jsx [hunk 2]
```diff
@@ -1,5 +1,74 @@
...
+    marginTop: "10px",
+  };
+
+  const imageWrapperStyle = {
+    borderRadius: "15px",
+    overflow: "hidden",
+  };
+
+  return (
+    <div style={containerStyle}>
```
Comment (pondohoo) [inline src/components/board/Member.jsx:41]: don't use inline css, use tailwind like you did below (use className)

File: src/components/board/Member.jsx [hunk 3]
```diff
@@ -1,5 +1,74 @@
...
+          </div>
+        </div>
+        <div className="flex flex-col items-start -mt-2">
+          <p className="text-2xl font-urbanist font-bold text-orange-500 pr-7 tracking-wide">
+            Jennie Kim
+          </p>
+          <p className="text-lg font-urbanist text-black pr-19">President</p>
+          <div className="flex">
+            <FaLinkedinIn className="h-6 w-6 text-orange-500" />
+            <IoMdMail className="h-6 w-6 text-orange-500 ml-2" />
```
Comment (pondohoo) [inline src/components/board/Member.jsx:61]: don't use default css colors, use our custom colors 
https://acm-ucr.github.io/docusaurus/docs/tailwind/colors/

File: src/components/board/Member.jsx [hunk 4]
```diff
@@ -1,5 +1,74 @@
...
+              src={placeholder}
+              alt="Member"
+              width={150}
+              height={150}
+              objectFit="cover"
+            />
+          </div>
+        </div>
+        <div className="flex flex-col items-start -mt-2">
+          <p className="text-2xl font-urbanist font-bold text-orange-500 pr-7 tracking-wide">
```
Comment (pondohoo) [inline src/components/board/Member.jsx:55]: don't use default css colors, use our custom colors
https://acm-ucr.github.io/docusaurus/docs/tailwind/colors/

File: src/components/board/Member.jsx [hunk 5]
```diff
@@ -1,5 +1,74 @@
...
+            Jennie Kim
+          </p>
+          <p className="text-lg font-urbanist text-black pr-19">President</p>
+          <div className="flex">
+            <FaLinkedinIn className="h-6 w-6 text-orange-500" />
```
Comment (pondohoo) [inline src/components/board/Member.jsx:60]: don't use default css colors, use our custom colors
https://acm-ucr.github.io/docusaurus/docs/tailwind/colors/

File: src/components/board/Member.jsx [hunk 6]
```diff
@@ -1,5 +1,74 @@
...
+            <IoMdMail className="h-6 w-6 text-orange-500 ml-2" />
+          </div>
+          <p className="italic text-xs font-urbanist text-black mt-1 -ml-1">
+            3rd year - Comp Sci
+          </p>
+          <p className="italic text-xs font-urbanist text-black mt-0.5 -ml-1">
```
Comment (pondohoo) [inline src/components/board/Member.jsx:66]: no need for italic

File: src/components/board/Member.jsx [hunk 7]
```diff
@@ -1,5 +1,74 @@
...
+        <div className="flex flex-col items-start -mt-2">
+          <p className="text-2xl font-urbanist font-bold text-orange-500 pr-7 tracking-wide">
+            Jennie Kim
+          </p>
+          <p className="text-lg font-urbanist text-black pr-19">President</p>
+          <div className="flex">
+            <FaLinkedinIn className="h-6 w-6 text-orange-500" />
+            <IoMdMail className="h-6 w-6 text-orange-500 ml-2" />
+          </div>
+          <p className="italic text-xs font-urbanist text-black mt-1 -ml-1">
```
Comment (pondohoo) [inline src/components/board/Member.jsx:63]: no need for italic

**Comment by syedmahdi30** [comment]
Screenshot_2
Fixed all the issues but let me know if I there is something wrong with the design

**Comment by syedmahdi30** [comment]
Screenshot_2

Removed the unnecessary padding

**Review by pondohoo — APPROVED** [review]

**Pull request merged**